CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+# Contributing to Spotlight
+
+Spotlight is a collaborative, open source project produced by developers, designers, product owners from several organizations. Spotlight mostly, uses the [contribution model](https://github.com/projectblacklight/blacklight/blob/master/CONTRIBUTING.md) from it's parent project [Blacklight](https://github.com/projectblacklight/blacklight). There are a few differences from the Blacklight process that this document will highlight.
+
+## User experience changes
+
+Spotlight has been designed by a team of user experience designers. While there are a lot of great opportunities to improve the user experience, some contributed changes might change other aspects of user experience that has been accounted. When contributing updates or changes to the Spotlight user experience here are some useful guidelines to follow.
+ - Determine if there is a way to contribute this in a backwards compatible or configurable way
+ - [Open an issue](https://github.com/projectblacklight/spotlight/issues/new) on the project and engage with core UX designers and developers
```

Comment:
> [Open an issue](https://github.com/projectblacklight/spotlight/issues/new) on the project ~~and~~ engage with core UX designers and developers

- Change "and" to "as a way to"
- Add "on your proposed change or update" to end of sentence

So:

- [Open an issue](https://github.com/projectblacklight/spotlight/issues/new) on the project as a way to engage with core UX designers and developers on your proposed change or update